source/beamline/BioXAS/BioXASFrontEndShutters.cpp

```diff
@@ -70,9 +70,9 @@ void BioXASFrontEndShutters::setDownstreamPhotonShutter(CLSExclusiveStatesContro
 AMAction3* BioXASFrontEndShutters::createMoveToOpenAction()
 {
 	AMListAction3 *actionList = new AMListAction3(new AMListActionInfo3("Opening front-end shutters", "Opening front-end shutters"), AMListAction3::Sequential);
-	actionList->addSubAction(createCheckChildIsOpenAction(upstreamPhotonShutter_, 10));
-	actionList->addSubAction(createMoveChildToOpenAction(safetyShutter_));
-	actionList->addSubAction(createMoveChildToOpenAction(downstreamPhotonShutter_));
+	actionList->addSubAction(createCheckChildIsOpen(upstreamPhotonShutter_, 10));
```

Comment:
Why the rename here? The 'action' part seems important, as that's what's actually returned.